pkgs/tools/admin/analog/default.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
   postPatch = ''
     sed -i src/anlghead.h \
       -e "s|#define DEFAULTCONFIGFILE .*|#define DEFAULTCONFIGFILE \"$out/etc/analog.cfg\"|g" \
-      -e "s|#define LANGDIR .*|#define LANGDIR \"$out/share/$pname}/lang/\"|g"
+      -e "s|#define LANGDIR .*|#define LANGDIR \"$out/share/$pname/lang/\"|g"
```

Comment:
```suggestion
      -e "s|#define LANGDIR .*|#define LANGDIR \"$out/share/${pname}/lang/\"|g"
```

Reply:
$name/${name} includes the version `something-0.0.1`
$pname/${pname} doesn't `something`

$pname is a shell variable
${pname} is replaced by nix before the phase is run

${} is easier to keep track of visually